bip-0002.mediawiki

```diff
@@ -65,12 +66,69 @@ If you are interested in assuming ownership of a BIP, send a message asking to t
 
 ===BIP Editors===
 
-The current BIP editor is Luke Dashjr who can be contacted at [[mailto:luke_bipeditor@dashjr.org|luke_bipeditor@dashjr.org]].
+The current BIP editor is Jaime Caring who can be contacted at [[mailto:jaime.caring@gmail.com|jaime.caring@gmail.com]].
+
+Jaime Caring is an interim editor and will not serve beyond June 1st, 2021.
+
+===BIP Editor Replacement and Stewardship===
+
+The following 23 GitHub users have been selected based on their contributorship
+to Bitcoin Core, current standing in the community, or commitment to the
+dissemination of high quality information as Stewards of the BIP process.
+
+* @achow101 Andrew Chow
+* @ajtowns Anthony Towns
+* @dongcarl Carl Dong
+* @Empact Ben Woosley
+* @gmaxwell Gregory Maxwell
+* @harding David A. Harding
+* @hebasto Hennadii Stepanov
+* @instagibbs Gregory Sanders
+* @jnewbery John Newbery
+* @jonasschnelli Jonas Schnelli
+* @jonatack Jon Atack
+* @kallewoof kallewoof
+* @laanwj W. J. van der Laan
+* @MarcoFalke MarcoFalke
```

Comment:
NACK. Please stop spamming this repository and the list of people here. I am not aware that anyone in this list consents to being included here. In fact there are several that wished to be taken off the list. I can't speak for others, but I certainly don't want to be involved in this and I don't want to further my responsibilities in Bitcoin apart from quality assurance where I voluntarily provide my feedback.